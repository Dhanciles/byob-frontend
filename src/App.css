
* {
  box-sizing: border-box;
  font-family: 'Helvetica', sans-serif;
}

.app-container {
  background: url('./images/tacos-2.jpg');
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.doc-header {
  width: 100%;
  background-color: #6a2800;
  color: #ffffff; 
  font-family: 'Fira Sans', sans-serif;
  min-height: 100px; 
  display: flex; 
  flex-direction: column;
  justify-content: space-around; 
  align-items: center;
  padding: 10px; 
}

.logo-text {
  margin: 0; 
  font-size: 3rem; 
}

.header-subtext {
  font-size: 1.5rem; 
}

h3 {
  margin: 20px 0 0 0;
} 

p {
  margin: 0;
}

pre {
  text-align: left;
  background: lightgray;
  padding: 20px;
  color: black;
}

.url-path {
  margin: 15px 0;
  font-weight: 800
}

.endpoint-container {
  display: flex;
  flex-direction: column;
  align-items: center;
  overflow: scroll;
  width: 100%;
}

.info-area {
  display: flex;
  flex-direction: column;
  padding: 50px;
}

.endpoint-item-container {
  background: #617200;
  width: 70%;
  display: flex;
  flex-direction: column;
  margin: 10px;
  color: white;
}

.endpoint-text-container {
  margin-left: 20px;
}

button {
  width: 200px;
  cursor: pointer; 
  font-weight: 700;
  font-size: 1rem;
}

button:hover {
  background-color: #364F5C; 
  color: #ffffff; 
}

.button-and-path-container {
  display: flex;
  justify-content: flex-start;
  background-color: #364F5C; 
}

.about {
  background: #a3d68f;
  width: 60%;
  margin: 50px;
  padding: 20px;
}

.about h2 {
  margin: 0 0 20px 0;
}
