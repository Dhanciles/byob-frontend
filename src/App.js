import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import { EndPointContainer }  from './EndPointContainer.js'


class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
        </header>
        <EndPointContainer />
      </div>
    );
  }
}

export default App;
