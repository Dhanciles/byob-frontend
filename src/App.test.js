import React from 'react';
import { shallow } from 'enzyme'; 
import App from './App.js'; 

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<App />)

    expect(wrapper).toMatchSnapshot()
  });

}); 

