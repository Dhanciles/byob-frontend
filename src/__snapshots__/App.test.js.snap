// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="doc-header"
        >
          <span
            className="logo-container"
          >
            <h1
              className="logo-text"
            >
              Taco Restaurants API
            </h1>
          </span>
          <h3
            className="header-subtext"
          >
            Find Top Taco Restaurants Across the US!
          </h3>
        </header>,
        <section
          className="about"
        >
          <h2>
            Welcome!
          </h2>
          <p>
            This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:
          </p>
          <p>
            <a
              href="https://kylie-joel-byob.herokuapp.com"
              target="_blank"
            >
              https://kylie-joel-byob.herokuapp.com
            </a>
            .
          </p>
          <p>
            Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!
          </p>
        </section>,
        <EndPointContainer />,
      ],
      "className": "app-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <span
              className="logo-container"
            >
              <h1
                className="logo-text"
              >
                Taco Restaurants API
              </h1>
            </span>,
            <h3
              className="header-subtext"
            >
              Find Top Taco Restaurants Across the US!
            </h3>,
          ],
          "className": "doc-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h1
                className="logo-text"
              >
                Taco Restaurants API
              </h1>,
              "className": "logo-container",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Taco Restaurants API",
                "className": "logo-text",
              },
              "ref": null,
              "rendered": "Taco Restaurants API",
              "type": "h1",
            },
            "type": "span",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Find Top Taco Restaurants Across the US!",
              "className": "header-subtext",
            },
            "ref": null,
            "rendered": "Find Top Taco Restaurants Across the US!",
            "type": "h3",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2>
              Welcome!
            </h2>,
            <p>
              This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:
            </p>,
            <p>
              <a
                href="https://kylie-joel-byob.herokuapp.com"
                target="_blank"
              >
                https://kylie-joel-byob.herokuapp.com
              </a>
              .
            </p>,
            <p>
              Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!
            </p>,
          ],
          "className": "about",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome!",
            },
            "ref": null,
            "rendered": "Welcome!",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:",
            },
            "ref": null,
            "rendered": "This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <a
                  href="https://kylie-joel-byob.herokuapp.com"
                  target="_blank"
                >
                  https://kylie-joel-byob.herokuapp.com
                </a>,
                ".",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "https://kylie-joel-byob.herokuapp.com",
                  "href": "https://kylie-joel-byob.herokuapp.com",
                  "target": "_blank",
                },
                "ref": null,
                "rendered": "https://kylie-joel-byob.herokuapp.com",
                "type": "a",
              },
              ".",
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!",
            },
            "ref": null,
            "rendered": "Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!",
            "type": "p",
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="doc-header"
          >
            <span
              className="logo-container"
            >
              <h1
                className="logo-text"
              >
                Taco Restaurants API
              </h1>
            </span>
            <h3
              className="header-subtext"
            >
              Find Top Taco Restaurants Across the US!
            </h3>
          </header>,
          <section
            className="about"
          >
            <h2>
              Welcome!
            </h2>
            <p>
              This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:
            </p>
            <p>
              <a
                href="https://kylie-joel-byob.herokuapp.com"
                target="_blank"
              >
                https://kylie-joel-byob.herokuapp.com
              </a>
              .
            </p>
            <p>
              Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!
            </p>
          </section>,
          <EndPointContainer />,
        ],
        "className": "app-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <span
                className="logo-container"
              >
                <h1
                  className="logo-text"
                >
                  Taco Restaurants API
                </h1>
              </span>,
              <h3
                className="header-subtext"
              >
                Find Top Taco Restaurants Across the US!
              </h3>,
            ],
            "className": "doc-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h1
                  className="logo-text"
                >
                  Taco Restaurants API
                </h1>,
                "className": "logo-container",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Taco Restaurants API",
                  "className": "logo-text",
                },
                "ref": null,
                "rendered": "Taco Restaurants API",
                "type": "h1",
              },
              "type": "span",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Find Top Taco Restaurants Across the US!",
                "className": "header-subtext",
              },
              "ref": null,
              "rendered": "Find Top Taco Restaurants Across the US!",
              "type": "h3",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                Welcome!
              </h2>,
              <p>
                This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:
              </p>,
              <p>
                <a
                  href="https://kylie-joel-byob.herokuapp.com"
                  target="_blank"
                >
                  https://kylie-joel-byob.herokuapp.com
                </a>
                .
              </p>,
              <p>
                Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!
              </p>,
            ],
            "className": "about",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Welcome!",
              },
              "ref": null,
              "rendered": "Welcome!",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:",
              },
              "ref": null,
              "rendered": "This API is designed to provide data on many of the best taco restaurants in cities all across the US. All of the endpoints below exist at the base URL:",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <a
                    href="https://kylie-joel-byob.herokuapp.com"
                    target="_blank"
                  >
                    https://kylie-joel-byob.herokuapp.com
                  </a>,
                  ".",
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "https://kylie-joel-byob.herokuapp.com",
                    "href": "https://kylie-joel-byob.herokuapp.com",
                    "target": "_blank",
                  },
                  "ref": null,
                  "rendered": "https://kylie-joel-byob.herokuapp.com",
                  "type": "a",
                },
                ".",
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!",
              },
              "ref": null,
              "rendered": "Please enjoy the data we've provided. If you're building an app or website that uses this data, let us know! We'd love to hear from you!",
              "type": "p",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
